src/components/Home.js

```diff
@@ -0,0 +1,65 @@
+import React, { useEffect } from 'react';
+import '../App.css';
+import { useSelector, useDispatch } from 'react-redux';
+import { getData, setCountry } from '../redux/reducer';
+import Card from './Card';
+
+const Home = () => {
+  const dispatch = useDispatch();
+  const data = useSelector((state) => state);
+  const test = data;
+  useEffect(() => {
+    const loadData = async () => {
+      await dispatch(getData());
+    };
+    if (data.length <= 0) {
+      loadData();
+    }
+  }, []);
+  const handleChange = () => {
+    const { value } = document.querySelector('.search-bar');
+    const result = test.filter((country) => (country.name).toLowerCase() === value);
+    if (result.length > 0) {
+      dispatch(setCountry(result));
+    }
+  };
+  const handleClear = () => {
+    const bar = document.querySelector('.search-bar');
+    bar.value = '';
+    dispatch(getData());
+  };
+  const showSearchBar = () => {
+    const searchSection = document.querySelector('.search-section');
+    searchSection.classList.toggle('hide');
+  };
+
+  return (
+    <main>
+      <div className="search-container">
+        <div className="top-header">
+          <h2>Covid 19 Stats</h2>
+          <i className="fas fa-search" aria-hidden="true" onClick={showSearchBar} />
+        </div>
+        <div className="main-header">
+          <img src="world-logo.png" alt="header-logo" className="header-logo" />
+          <p className="header-text">Here you&apos;ll find the real live information about covid 19 sorted by countries.</p>
+        </div>
```

Comment:
Kindly look at the images below.
- [ ] You did well by adding the missing header (first image). However, you still need to improve on the design to look similar to what we have on the template (second image). Please add appropriate spacing between the image and paragraph so that they are in different halves of the div like we have in the template. You can use CSS grid or flexbox for this.
- [ ] Also, kindly reduce the size of your header image a bit and make your text larger since it is a header.

![lassis-jumbo](https://user-images.githubusercontent.com/36542986/142460314-df00f4b6-f7b3-4c68-89db-5d60f3f00133.jpg)

Reply:
This issue is well resolved! ✔️